ada_feeding/config/ada_planning_scene.yaml

```diff
@@ -86,8 +86,8 @@ ada_planning_scene:
       touch_links: [forkTine] # the links, in addition to frame_id, that should be ignored for collision checking
     # The offset values for the center coordinates ([x, y, z]) of the table. These 
     # values are used to translate the detected position into the table's origin.
-    table_detection_offsets: [-0.20, -0.25, -0.79]
-    # A threshold on the distance between the default table quaternion 
+    table_detection_offsets: [-0.20, -0.25, -0.735]
+    # A threshold on the angular distance between the default table quaternion 
```

Comment:
Nit: both for this parameter and the above one, mention units in the comment. I know the position offsets are in meters. Is the angular distance threshold in radians?